### PR TITLE
Django 4.2 support

### PR DESCRIPTION
**Pull request by salman2013**
1. Add django42 support.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -21,17 +21,17 @@ whitelist_externals =
 deps =
     django31: Django>=3.1,<3.2
```
Comment (mumarkhan999) [inline tox.ini:22]: I guess, this line should be removed as well.

Comment (salman2013) [inline tox.ini:22]:
```suggestion
```

File: tox.ini [hunk 2]
```diff
@@ -19,19 +19,17 @@ whitelist_externals =
     mkdir
     rm
 deps =
-    django31: Django>=3.1,<3.2
-    django32: Django>=3.2,<4.0
```
Comment (mumarkhan999) [inline tox.ini:23]: We only need to remove `django31: Django>=3.1,<3.2`.

`django32` shouldn't be removed.

Am I right?

File: tox.ini [hunk 3]
```diff
@@ -19,19 +19,17 @@ whitelist_externals =
...
     BOKCHOY_HEADLESS
 setenv =
     DJANGO_SETTINGS_MODULE = workbench.settings
     SCREENSHOT_DIR={toxinidir}/var/logs
     SELENIUM_DRIVER_LOG_DIR={toxinidir}/var/logs
-commands = 
+commands =
     mkdir -p {toxinidir}/var/logs
     rm -rf {toxinidir}/var/logs/*
-    pytest {posargs}
```
Comment (mumarkhan999) [inline tox.ini:34]: Is it intentional?

Comment (salman2013) [reply tox.ini:34]: @awais786 make this change when fixing test cases.

File: requirements/travis.txt [hunk 4]
```diff
@@ -1,69 +1,74 @@
...
 idna==3.4
     # via requests
-packaging==21.3
+packaging==23.1
     # via tox
-path==16.5.0
+path==16.7.1
     # via edx-i18n-tools
-platformdirs==2.5.2
+<<<<<<< HEAD
```
Comment (mumarkhan999) [inline requirements/travis.txt:38]: ⚠️⚠️⚠️ Conflict here.

**Review by mumarkhan999 — CHANGES_REQUESTED** [review]

**Review by awais786 — APPROVED** [review]

**Review by mumarkhan999 — APPROVED** [review]

**Pull request merged**